### DISCUSSION TITLE
Антизапрет + свой SOCK5 прокси для обхода тормозов в ютуб

### DISCUSSION THREAD

**Discussion by Kor-Phaeron** [anticensority/runet-censorship-bypass#290] [Q&A]
Добрый день! Подскажите, пожалуйста, заметил, что можно добавлять свои прокси. На своем сервере сделал прокси для обхода замедления ютуба, если использовать, допустим, фаерфокс, где конкретно есть настройка прописывания прокси - ютуб летает. При использовании эджа (чем всегда и пользуюсь и где стоит ваше расширение) и указании своего прокси SOCK5 - ютуб как был мертвым так и остается.

Если отключить ваше расширение и через другое расширение "Socks5 Configurator" для эджа прописать адрес моего сервера, то ютуб снова летает, но, разумеется, ваше расширение ругается на настройки прокси из другого и не включается. Одновременно они работать не хотят и переключаться то с одного, чтобы зайти на рутрекер, то на другой, чтобы зайти на ютуб такое себе.

Может быть я что-то неверно делаю? С точно такими же настройками (как ниже из другого расширения) ютуб работает великолепно.

![image](https://github.com/user-attachments/assets/23244ffd-1372-4b46-a31f-ea9136ce152d)

**Comment by villabajo** [comment]
тот же вопрос в связи с использованием spoofDPI на MacOS

**Comment by ilyaigpetrov** [comment]
Удостоверьтесь, что вы прописали в исключения `*.googlevideo.com` и `*.youtube.com`.
`Иконка синей ленты` -> `Исключения / Exceptions` -> Две встречные стрелочки справа.

**Comment by villabajo** [comment]
прописано, но главная ютюба не грузится, пока не вырубишь расширение. они же оба прокси, и расширение и spoofDPI, работает либо то, либо другое. т.е. я так полагаю надо не просто исключение прописывать, а "использовать СВОИ прокси
СВОИ прокси только для СВОИХ сайтов" -- но это тоже не работает
